AWSSDKSwiftCLI/Tests/AWSSDKSwiftCLITests/Models/ReleaseNotesBuilderTests.swift

```diff
@@ -12,7 +12,7 @@ import XCTest
 /*
  *  Regression tests for protection against change in generated release notes markdown content.
  */
-class ReleaseNotesBuilderTests: XCTestCase {
+class ReleaseNotesBuilderTests: CLITestCase {
```

Comment:
This test now inherits from CLITestCase, which sets up a temporary directory structure that allows the Trebuchet files to be written to the workspace directory.